cmd/kyma/alpha/create/module/moduleconfig.go

```diff
@@ -0,0 +1,149 @@
+package module
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/blang/semver/v4"
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
+	Name          string            `yaml:"name"`         //required, the name of the Module
+	Version       string            `yaml:"version"`      //required, the version of the Module
+	Channel       string            `yaml:"channel"`      //required, channel that should be used in the ModuleTemplate
```

Comment:
I configed a ymal file with `fast` channel, however, the generated template.yaml have `channel: regular`

Reply:
Nice catch, fixed.